### PR TITLE
Add more "influencer" links

### PR DESCRIPTION
**Pull request by msaleh75894**
Instead on just one website that offer hiring influencers 
I have added 3 options to choose from

#### Summary
Instead on just one website that offer hiring influencers 
I have added 3 options to choose from

#### Motivation
This is going to help readers choose from more than one  serivce 

#### Supporting details
links included

### CODE REVIEW

**Comment by estelle** [comment]
My take is that all services should be removed instead of new ones added. MDN is not a place to market marketing services.

**Review by estelle — CHANGES_REQUESTED** [review]
I suggest removing all three. MDN is not a place to market marketing services.

**Comment by Rumyra** [comment]
Hi @msaleh75894 thanks for your contribution! It's correct we are trying to minimise the number of external links on MDN currently.

A good update would be to remove the whole Youtube paragraph - it's out of date - and instead add a note about youtube and twitch to the social media paragraph 👍

File: files/en-us/games/publishing_games/game_promotion/index.html [hunk 1]
```diff
@@ -62,7 +62,7 @@ <h2 id="YouTubers">YouTubers</h2>
...
+<p>There are two options to get coverage from the YouTubers: first is you contact them directly and send a link to your game via email or private message. The second is earned in time - if you're known enough the YouTubers will contact you and ask for the link or build of your game. This <a href="https://docs.google.com/spreadsheets/d/1MVrNBLVkL-VzKe4pFHHVx0bfCdInB3G2FVJbc9V4CUk/edit#gid=0">Big List of YouTubers</a> is a great place to start. You can find YouTube and Twitch.tv influencers at <a href="https://gameinfluencer.com/">gameInfluencer.com</a>, <a href="https://syft.gg/">syft.gg</a> or <a href="https://hireinfluence.com/">hireinfluence.com</a> to help promote your game.</p>
```
Comment (wbamberg) [inline files/en-us/games/publishing_games/game_promotion/index.html:65]:
```suggestion
```

Comment (wbamberg) [inline files/en-us/games/publishing_games/game_promotion/index.html:65]:
```suggestion
```

**Comment by Rumyra** [comment]
Due to the conflicts I'm going to suggest we close this, open an issue to remove the paragraph and tackle it after md conversion. Let me know what you think @wbamberg

**Comment by wbamberg** [comment]
> Due to the conflicts I'm going to suggest we close this, open an issue to remove the paragraph and tackle it after md conversion. Let me know what you think @wbamberg

I agree and have filed https://github.com/mdn/content/issues/9831.

**Pull request closed without merging**